homeassistant/components/yolink/config_flow.py

```diff
@@ -0,0 +1,34 @@
+"""Config flow for yolink."""
```

Comment:
I didn't look at your PR yet, however you will need to add a test for your Config Flow with 100% coverage.

Reply:
thank you. I'll do this soon